checkov/sca_package/scanner.py

```diff
@@ -127,18 +127,30 @@ def run_scan_busy_wait(self, input_path: Path, scan_id: str) -> dict[str, Any]:
                 logging.error(response_json["outputData"])
                 return {}
 
+            time.sleep(SLEEP_DURATION)
+            total_sleeping_time += SLEEP_DURATION
+
             if total_sleeping_time > MAX_SLEEP_DURATION:
                 logging.info(f"Timeout, slept for {total_sleeping_time}")
                 return {}
 
-            time.sleep(SLEEP_DURATION)
-            total_sleeping_time += SLEEP_DURATION
+        # handle license violations
+        request_body = {
+            "s3ObjectKey": scan_id
+        }
+        license_data = request_wrapper(
+            "POST", f"{self._base_url}/api/v1/vulnerabilities/packages/license-violations",
+            headers=bc_integration.get_default_headers("POST"),
+            json=request_body
+        )
+        license_data_json = license_data.json()
```

Comment:
can the response be an empty dict/json? if yes, then the next line will blow up.

Reply:
@gruebel  the default response in case there's no data is {"packages": []} so should not crash in this case.